tests/Testcontainers.LocalStack.Tests/LocalStackContainerTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections.Generic;
+using Amazon.CloudWatchLogs;
+using Amazon.CloudWatchLogs.Model;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Amazon.Runtime;
+using Amazon.SimpleNotificationService;
+using Amazon.SimpleNotificationService.Model;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+
+namespace Testcontainers.Minio;
+
+public sealed class LocalStackContainerTest : IAsyncLifetime
+{
+    private readonly LocalStackContainer _localStackContainer = new LocalStackBuilder()
+        .WithServices(AwsService.S3, AwsService.DynamoDb, AwsService.CloudWatchLogs, AwsService.Sqs, AwsService.Sns)
```

Comment:
xUnit.net will spin up a container for each `[Fact]`. Is it faster (better) to spin up services individually? E.g. the S3 test does not require DynamoDb, or doesn't that matter because it is quick anyway?

Reply:
In newer local stack it doesn't matter because all services are enabled by default
My mistake was that i thought it works the same as in 2019